permissions/Videos.permission

```diff
@@ -1,7 +1,7 @@
 # -*- mode: sh -*-
 
-# x-sailjail-translation-catalog =
-# x-sailjail-translation-key =
+# x-sailjail-translation-catalog = sailjail-permissions
+# x-sailjail-translation-key = permission-la-videos
 # x-sailjail-description = Access Video data
```

Comment:
I find these names with capital letters strange. In this instance it doesn't even mean "Video" folder but "Videos". Would it be better if this was just "Access video data" or "Access videos" even. Then again it applies to only those that are in Videos folder. If there was secondary description that could explain it too. (Also how to access those on memory card..?)

Reply:
Noticed similar errors on other descriptions. Will amend those a bit e.g. this should be just "Access videos" (without data)